tests/test_treepoem.py

```diff
@@ -13,6 +13,7 @@
 @pytest.mark.parametrize('barcode_type,barcode_data', [
     ('qrcode', "This is qrcode barcode."),
     ('azteccode', "This is azteccode barcode."),
+    ('azteccode', b"This is azteccode barcode."),
```

Comment:
In what way do they fail? Are you running `tox` ?

Reply:
Yes, I am running `tox` in a virtualenv. Every barcode test ends in:

```
        # Trying to prevent a `ResourceWarning`.
        # Bug: https://github.com/python-pillow/Pillow/issues/1144
        # Workaround: https://github.com/python-pillow/Pillow/issues/835
        with open(fixture_path, 'rb') as fixture:
            expected = Image.open(fixture)
            bbox = ImageChops.difference(actual, expected).getbbox()
>           assert bbox is None
E           assert (0, 1, 798, 146) is None

tests/test_treepoem.py:42: AssertionError
```

I have no clue where to look for the cause.

Reply:
It's the version of ghostscript - I now have GS 9.23 and they fail. Will regenerate later

Reply:
@jcassee I made #124 for the test fixture failure. If you have a spare minute, can you try recreating the fixtures with the below patch, and seeing if they match what I'm seeing, and post on the issue?

```patch
diff --git a/tests/test_treepoem.py b/tests/test_treepoem.py
index f1f7a94..a593cff 100644
--- a/tests/test_treepoem.py
+++ b/tests/test_treepoem.py
@@ -32,7 +32,7 @@ def test_barcode(barcode_type, barcode_data):
     )
 
     # Uncomment to rebuild fixtures:
-    # actual.save(fixture_path)
+    actual.save(fixture_path)
 
     # Trying to prevent a `ResourceWarning`.
     # Bug: https://github.com/python-pillow/Pillow/issues/1144
```